README.md

```diff
@@ -67,12 +67,44 @@ An example of metadata file is the following:
 ```
 This metadata file specifies two sound event ids: `framework/drumkit/0` and `framework/piano/0`. Then each entry, tells what is the location of the audio file (`sound-file`), the license of the sound (`license`) and the source where it was obtained from (`source`).
 
+## A full example
+```
+{
+    "example/sound/1": {
+        "sound-file": "sound0.wav",
+        "sound-files": [
+            "sound1.wav",
+            "sound2.wav",
+            "sound3.wav"
+		]
+        "loop": true,
+        "fade-in": 3000,
+        "fade-out": 5000,
+        "volume": 0.5,
+        "pitch": 2.0,
+        "delay": 10000,
+        "overlap-behavior": "restart"
+    }
+```
+This example shows all the options that the metadata file accepts.
+
+- **`sound-file`**: Indicates the path to the sound file to be played.
+- **`sound-files`**: It's an array of paths, and indicates that one of these sounds should be picked up randomly to be played. If like in this example, `sound-file` and `sound-files` are set, then all the specified paths will be considered, in other words, for this example, one sound among "sound0.wav", "sound1.wav", "sound2.wav" and "sound3.wav" would be played.
+- **`loop`**: If set to `true` the sound will be played again when it finishes. *Defaults to `false`*.
+- **`fade-in`**: Indicates the time duration in which the volume of the sound should fade in from the start. The used unit is milliseconds. *Defaults to 1000 only if `loop` is set to `true`.*
+- **`fade-out`**: Indicates the time duration in which the volume of the sound should fade out after the sound is stopped. *The used unit is milliseconds. Defaults to 1000 only if `loop` is set to `true`.*
+- **`volume`**: Indicates the volume level the sound you play at. *Defaults to 1.0 (the "normal" volume).*
+- **`pitch`**: Sets the sound pitch while keeping the original tempo (speed). *Defaults to 1.0 (the "normal" pitch).
+- **`rate`**: Sets the tempo and pitch. *Defaults to 1.0 (the "normal" pitch*.
```

Comment:
Forgot to close parenthesis, also, "the normal pitch"? wouldn't be "the normal rate/tempo and pitch"?